.github/workflows/tests.yml

```
@@ -0,0 +1,13 @@
+name: Runs tests
+on: [push]
+jobs:
+    Explore-GitHub-Actions:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Check out repository code
+              uses: actions/checkout@v2
+            - run: npm install
```

```suggestion
            - run: npm ci
```
this works too and is a bit faster :+1: